src/server/web_server.js

```diff
@@ -285,6 +286,27 @@ app.get('/version', (req, res) => getVersion(req.url)
     })
 );
 
+
+// Get the NSFS stats
+app.get('/metrics/nsfs_stats', async (req, res) => {
+    const { report, status } = _create_nsfs_report();
+    res.send(report);
+    if (status !== 200) res.status(status);
+    res.end();
```

Comment:
maybe just use res.status(200).end(); like we do in all the other parts in the code?
```suggestion
    res.status(200).end();
```
and remove all of the other status lines

Reply:
👍